add case studies instruction (#442)
kubeedge/website: docs/community/casestudies.md

1.  At the beginning of the document, please include the following information. This information will be displayed on the KubeEdge official website's case studies homepage in a card format, allowing users to quickly understand the general content of the case.

```
title: Company Name
subtitle：(Optional) Please use keywords to describe the significant effect of the case study. (Example: 100% Performance Improvement)
description： Briefly summarize the case study in one sentence.
date： Date (Format: 2023-01-01)
tags： Case tags. Please select Solution or End User based on the type of case.
```

2.  Main Content: Please provide a detailed description of the case study in the main content of the document. You may consider the following points for detailed exposition:

- Case title
- Case background
- Application scenario
- Challenges faced
- Why choose KubeEdge
- Solution and technical architecture
- Final outcome and realized value
- Future plans

# Submit Your Case

> After completing your case study write-up, please submit your case document to the KubeEdge [website](https://github.com/kubeedge/website) repository, and it will be displayed in the official website's case studies page after community review.

Reviewer: ```suggestion
After completing your case study write-up, please submit your case document to the KubeEdge [website](https://github.com/kubeedge/website) repository, you can submit your case in English or Chinese(languages ​​currently supported by the website), and it will be displayed in the official website's case studies page after community review. 
```

> 1.  Create a directory for your case study under `src/pages/case-studies` and `i18n/zh/docusaurus-plugin-content-page/case-studies`, naming the directory after your case.

Reviewer: ```suggestion
1.  Create a directory for your case study under `src/pages/case-studies` in English or `i18n/zh/docusaurus-plugin-content-page/case-studies` in Chinese, naming the directory after your case.
```

> 2.  Place your English case document under the directory you created in `src/pages/case-studies/`, and place your Chinese case document under the corresponding directory you created in `i18n/zh/docusaurus-plugin-content-page/case-studies`.

Reviewer: ```suggestion
2.  Place your English case document under the directory you created in `src/pages/case-studies/`, or place your Chinese case document under the corresponding directory you created in `i18n/zh/docusaurus-plugin-content-page/case-studies`.
```
